src/components/LeafletMap.js

```diff
@@ -269,4 +269,10 @@ export class LeafletMap {
       dispatch.call("hide-loading-indicator");
     }
   };
+  findAutocompletLocation(bbox) {
```

Comment:
Maybe consider adding a marker to the map for the lat lon location returned by the geocoder, rather than the bbox. I think adding a marker to the map might be a better UX as it gives the user a clear indication of what they searched for on the map. You can also have the marker display its pop-up when it is added, with the autocomplete result text inside.

Reply:
Adds a marker now